.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Please feel free to remove whatever sections/lines in this aren't relevant. -->
+
+## Title Line Template: [Website] - [UI components]: [Brief description.]
```

Comment:
What is [Website]?

Maybe [UI components] should be singular since an issue would be about a single item, i.e. [UI component] cc @kategarklavs